### PR TITLE
Move to ghcr registry

### PR DESCRIPTION
**Pull request by mpfeil**
### Environment
- OS: Write here
- Browser & Version: Write here

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ on:
     types:
       - published
 
+env:
+  REGISTRY: ghcr.io
+
 jobs:
   lint:
     name: Lint code using eslint
```
Comment (ubergesundheit) [inline .github/workflows/build.yaml:21]:
```suggestion
    name: Lint code using eslint
    permissions:
      contents: read
      packages: write
```

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -110,11 +113,12 @@ jobs:
...
 
-      - name: Login to DockerHub
+      - name: Log in to Github Container Registry (ghcr)
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKER_HUB_USER }}
-          password: ${{ secrets.DOCKER_HUB_PASS }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GHCR_TOKEN }}
```
Comment (ubergesundheit) [inline .github/workflows/build.yaml:121]:
```suggestion
          password: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/build.yaml (hunk 1 above)
Comment (ubergesundheit) [reply .github/workflows/build.yaml:21]: no need for a separate GitHub token in secrets

Comment (mpfeil) [reply .github/workflows/build.yaml:21]: A seperate GitHub token is needed for running the `prune` actions. If this workflow runs with the standard one nothing will happen and all untagged images must be delete by hand.

Check this one here: https://github.community/t/trigger-on-package-registry-publish-not-working/189656/2

**Pull request merged**